packages/list-view/tests/list_view_test.js

```diff
@@ -56,13 +58,15 @@ test("should render an empty view when there is no content", function() {
         style: 'height:' + emptyViewHeight + 'px;'
       });
 
-  view = Ember.ListView.create({
-    content: content,
-    height: height,
-    rowHeight: rowHeight,
-    itemViewClass: itemViewClass,
-    emptyView: emptyView
-  });
+  Ember.run(function(){
+    view = Ember.ListView.create({
```

Comment:
`Ember.ListView.create` requires run loop. I've seen this before, I'm not sure why it worked before.